Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:31 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:31 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:31 /myfile3
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:31 /myfile4
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:31 /myfile5
file 1 has been put into HDFS
file 1 has been read from HDFS for 1 time
diff succeed!
file 1 has been read from HDFS for 2 time
diff succeed!
file 1 has been read from HDFS for 3 time
diff succeed!
file 1 has been read from HDFS for 4 time
diff succeed!
2019-03-27 06:33:10,641 INFO  retry.RetryInvocationHandler RetryInvocationHandler.java:log:411 - java.net.ConnectException: Call From clnode051.clemson.cloudlab.us/130.127.133.60 to node-1-link-0:8020 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused, while invoking ClientNamenodeProtocolTranslatorPB.getFileInfo over node-1-link-0/10.10.1.4:8020 after 1 failover attempts. Trying to failover after sleeping for 1473ms.
file 1 has been read from HDFS for 5 time
diff succeed!
file 1 has been read from HDFS for 6 time
diff succeed!
file 1 has been read from HDFS for 7 time
diff succeed!
file 1 has been read from HDFS for 8 time
diff succeed!
file 1 has been read from HDFS for 9 time
diff succeed!
file 1 has been read from HDFS for 10 time
diff succeed!
Deleted /myfile1
file 1 has been removed from HDFS
Found 5 items
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:33 /myfile1
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:33 /myfile2
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:33 /myfile3
-rw-r--r--   3 root supergroup  268435456 2019-03-27 06:34 /myfile4._COPYING_
-rw-r--r--   3 root supergroup 1073741824 2019-03-27 06:34 /myfile5
file 1 has been put into HDFS
file 1 has been read from HDFS for 1 time
diff succeed!
